docs/extensions/guides/README.md

```diff
@@ -19,10 +19,11 @@ Each guide or sample will include:
 | [Stores](stores.md) | |
 | [Components](components.md) | |
 | [KubeObjectListLayout](kube-object-list-layout.md) | |
+| [Working with mobx](working-with-mobx.md) | |
```

Comment:
This probably should be included in menu/navigation?

Reply:
I don't understand. I thought that is what this was doing.

Reply:
I mean this: https://github.com/lensapp/lens/blob/master/mkdocs.yml#L31-L35

Reply:
Then yes, will do.